ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/ConnectIntegrationTest.java

```diff
@@ -326,6 +363,21 @@ public void shouldWriteTimestampsToConnect() throws UnsupportedEncodingException
         stringContainsInOrder(sinkOutputParts), TIMEOUT_NS, TimeUnit.NANOSECONDS);
   }
 
+  @Test
+  public void shouldListConnectorPlugins() {
+    // Given:
+    create("mock-connector", ImmutableMap.of(
+        "connector.class", "org.apache.kafka.connect.tools.MockSourceConnector"));
+
+    // When:
+    final RestResponse<KsqlEntityList> response = ksqlRestClient.makeKsqlRequest("LIST CONNECTOR PLUGINS;");
+
+    // Then:
+    assertThat(response.isSuccessful(), is(true));
+    assertThat(response.getResponse().get(0), instanceOf(ConnectorPluginsList.class));
+    assertThat(((ConnectorPluginsList)response.getResponse().get(0)).getConnectorsPlugins().size(), is(0));
```

Comment:
Do you know why there are no connector plugins listed here? It'd be good to try to understand/see if we can add one, or at minimum add a comment to explain why it is expected that none are returned from this test.